setup.py

```diff
@@ -15,6 +15,7 @@
     'easy-thumbnails',
     'phonenumbers',
     'vobject',
+    'sphinx-rtd-theme',
```

Comment:
Here https://github.com/aldryn/aldryn-people/blob/docs/test_requirements/base.txt#L10 would be a better place.